client/state/rewind/size/test/reducer.js

```diff
@@ -82,4 +82,30 @@ describe( 'rewind.size reducers', () => {
 			).toEqual( 7 );
 		} );
 	} );
+	describe( 'backupsStopped', () => {
+		it( 'should return false when there is nothing in the state', () => {
+			expect(
+				sizeReducer( undefined, { type: REWIND_SIZE_SET, size: {} } ).backupsStopped
```

Comment:
Is it valid to dispatch `REWIND_SIZE_SET` with an empty `size` object?
I understand this action is dispatched when the `/rewind/size` endpoint returns 200.

If there's a chance that the endpoint might return 200 and don't include the `backupsStopped` flag, I think it's still okay to test it.